pkg/controllers/resources/ingressclasses/syncer.go

```diff
@@ -82,3 +88,11 @@ func (i *ingressClassSyncer) SyncToHost(ctx *synccontext.SyncContext, event *syn
 	ctx.Log.Infof("delete virtual ingress class %s, because physical object is missing", event.Virtual.Name)
 	return ctrl.Result{}, ctx.VirtualClient.Delete(ctx, event.Virtual)
 }
+
+func (i *ingressClassSyncer) ExcludeVirtual(_ client.Object) bool {
+	return true
```

Comment:
Why is this true?

Reply:
Because (from what I know) we don't sync `ingressClasses` toHost.
But anyways I'll turn it back to `false`.